jpegoptim.c

```diff
@@ -806,14 +810,14 @@ int optimize(FILE *log_fh, const char *filename, const char *newname,
 			cinfo.arith_code = (arith_mode > 0 ? TRUE : FALSE);
 #endif
 		if (dinfo.saw_Adobe_marker && (save_adobe || strip_none)) {
-			/* If outputting Adobe marker, dont write JFIF marker... */
+			/* If outputting Adobe marker, don't write JFIF marker... */
 			cinfo.write_JFIF_header = FALSE;
 		}
 
 		j=0;
-		jpeg_start_compress(&cinfo,TRUE);
+        jpeg_start_compress(&cinfo,TRUE);
```

Comment:
Indentation seems off in this patch

Reply:
Thanks, accidentally used spaces instead of tabs.